code/workspaces/auth-service/resources/permissions.yml

```diff
@@ -3,6 +3,12 @@ instanceAdmin:
   - name: instance
     permissions:
       - admin
+  - name: projects
```

Comment:
You shouldn't need this block. Short version is that the permission `system:instance:admin` will be all powerful and Josh has implemented this already in the GraphQL layer. We probably need to extend the permission checking to support this in the infrastructure api and auth service